portal-kernel/src/com/liferay/portal/kernel/servlet/MetaInfoCacheServletResponse.java

```diff
@@ -454,6 +454,17 @@ public void setContentLength(int contentLength) {
 		super.setContentLength(contentLength);
 	}
 
+	@Override
+	public void setContentLengthLong(long contentLength) {
+		if (isCommitted()) {
+			return;
+		}
+
+		_metaData._contentLength = contentLength;
+
+		super.setContentLengthLong(contentLength);
+	}
+
```

Comment:
@holatuwol I am surprised that the addition of the `setContentLengthLong(long)` method did not cause a Semantic Version MAJOR change since package com.liferay.portal.kernel.servlet is exported. You might need to add a `@ProviderType` annotation.